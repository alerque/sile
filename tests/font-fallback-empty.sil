\begin[papersize=a6]{document}
\nofolios
\noindent
\use[module=packages.font-fallback]
\font:add-fallback[family=Noto Serif CJK JP]
\font:remove-fallback
ã‚“
\end{document}
