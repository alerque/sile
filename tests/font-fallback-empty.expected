Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	29.7654
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	0 w=6.8359	(ã‚“)
End page
Finish
